cli/src/command/account.rs

```diff
@@ -992,3 +1002,31 @@ async fn print_address(account: &Account, address: &AccountAddress) -> Result<()
 
     Ok(())
 }
+
+async fn print_outputs(mut outputs: Vec<OutputData>, title: &str) -> Result<(), Error> {
+    outputs.sort_by(|a, b| {
+        a.metadata
+            .milestone_timestamp_booked()
+            .cmp(&b.metadata.milestone_timestamp_booked())
+    });
+
+    if outputs.is_empty() {
+        println_log_info!("No outputs found");
+    } else {
+        println_log_info!("{title}");
+        for (i, output_data) in outputs.into_iter().rev().enumerate() {
+            let booked_time = to_utc_date_time(output_data.metadata.milestone_timestamp_booked() as u128 * 1000)?;
+            let formatted_time = booked_time.format("%Y-%m-%d %H:%M:%S").to_string();
```

Comment:
```suggestion
            let formatted_time = booked_time.format("%Y-%m-%d %H:%M:%S UTC").to_string();
```
Otherwise it may be a bit confusing to people if they have a different value than the explorer idk

Reply:
right ... should I change it for the transaction as well?

Reply:
Yeah

Reply:
Wait actually I don't know if you can put this directly in the format thing, or if you'll have to put it later

Reply:
all good

Reply:
Included this suggestion also for the `transactions` command, that's why I did not merge the suggestion.